src/utils/renderers.js

```diff
@@ -35,6 +35,16 @@ export const headingRenderer = ({ children: reactChildren, level }) => {
   return React.createElement("h" + level, { id: slug }, children);
 };
 
+export const absoluteImageRenderer = ({ src, alt }) => {
+  src = src.replace("./", "http://localhost:9000/")
```

Comment:
There is a variable in the book info that takes care of the local URL, I am using it here: https://github.com/SaraVieira/starter-book/blob/4d0970f32c918653fbc34e8295f588db60fd3f3a/generate-book/pdf.js#L12-L13

Mind using that?